ako-operator/controllers/role.go

```diff
@@ -142,6 +147,11 @@ func BuildClusterrole(ako akov1alpha1.AKOConfig, r *AKOConfigReconciler, log log
 				Resources: []string{"gateways", "gateways/status", "gatewayclasses", "gatewayclasses/status"},
 				Verbs:     []string{"get", "watch", "list", "patch", "update"},
 			},
+			{
```

Comment:
same here

Reply:
Actually, release-1.9.3 was branched out from master before release-1.9.1-operator changes could be cherry-picked back to master. Hence, release-1.9.1-operator changes are not present in release-1.9.3-operator(which was branched out from release-1.9.3).